.github/workflows/e2e_tests.yaml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches:
       - staging
-      - feat-e2e-tests
+      - e2e-**
```

Comment:
@elbotho I would suggest added a comment like this:

`# even if tests run on a different branch, they are hard coded to the staging deployment`